apps/broker/test/mqtt/session_test.exs

```diff
@@ -3,7 +3,8 @@ defmodule Mqtt.SessionTest do
   alias Mqtt.Session
 
   setup do
-    :mnesia.clear_table(Session)
+    # TODO: why does this always return `{:aborted, {:no_exists, Session}}`?
```

Comment:
I believe that will return something else if there is actually a table to clear. Not sure how to clear the table in a re-entrant sort of way without ignoring the response.

Something same/similar happens when tables are created on startup